deep_qa/data/instances/tuple_inference_instance.py

```diff
@@ -329,13 +349,16 @@ def get_lengths_from_indexed_tuples(indexed_tuples: List[List[List[int]]]):
     @overrides
     def pad(self, max_lengths: Dict[str, int]):
         """
-        Pads (or truncates) each indexed tuple in answers_indexed and background_indexed to specified lengths,
-        using the superclass methods ``pad_sequence_to_length`` (for padding number of options, number of tuples,
-        and number of slots per tuple) and ''pad_word_sequence`` (for padding the words in each slot), with the
-        following exception:
-            When padding the number of answer options, if there are more answers provided, here we don't
-            remove options, we raise an exception because we don't want to risk removing the correct answer in
-            a multiple choice setting.
+        Pads (or truncates) each indexed tuple in answers_indexed and
+        background_indexed to specified lengths, using the superclass methods
+        ``pad_sequence_to_length`` (for padding number of options, number of
+        tuples, and number of slots per tuple) and ''pad_word_sequence`` (for
```

Comment:
indeed, i didn't check the contents of a lot of docstrings and just reformatted them to be compliant